app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] You're required to ask a new registered user to confirm his or her email before he or she would be able to login in. This feature is missing in your device setup. It should look something like this. 

```
  	devise :database_authenticatable, :registerable,
	:recoverable, :rememberable, :validatable,  :confirmable

```